### PR TITLE
Day 4

### PR DESCRIPTION
**Pull request by mahdimahdawi**
**This PR Contains:**
- Created a table by the name of vets with proper fields.
- Created a link table between vets and species by the name of specialization and set vet_id and species_id foreign keys and references to vets and species tables.
- Created a link table between animals and vets tables and set foreign key also references to both tables.
- The given data is inserted to vets, specialization, and visits tables.
- Queries runed in local and added the queries on queries.sql file.
- The result of the queries are attached to this PR.

-

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -42,3 +42,42 @@ CREATE TABLE species(
...
+FOREIGN KEY(vet_id)
+REFERENCES vets(id);
+
+ALTER TABLE specialization
+ADD CONSTRAINT species_fk
+FOREIGN KEY(species_id)
+REFERENCES species(id);
+
+CREATE TABLE visits(
+    animal_id INT,
```
Comment (EricMbouwe) [inline schema.sql:71]: - [x] There is one field missing here, the `visit_date date ` to keep a record of the visit date.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @mahdimahdawi, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
-  Data ✔
- Queries ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahdimahdawi** [comment]
REQUIRED CHANGES DONE

**Review by elmar8287 — APPROVED** [review]
Hi, @mahdimahdawi.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- Good PR description 👍 

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**